share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-x86_64_v4/spack.yaml

```diff
@@ -117,6 +123,8 @@ spack:
       externals:
       - prefix: /opt/slurm
         spec: slurm@22.05.8 +pmix
+      require:
+      - +pmix
```

Comment:
why?

Reply:
Externals don't have dependencies. In `develop` this would not be an issue, but in the branch having `%gcc` under default requirements prevents externals to be selected.

Reply:
I can keep it in the branch, if that's preferred.

Reply:
Yeah, that would make more sense.